pkg/traceql/parse.go

```diff
@@ -22,7 +22,7 @@ func Parse(s string) (expr *RootExpr, err error) {
 		if r := recover(); r != nil {
 			var ok bool
 			if err, ok = r.(error); ok {
-				if errors.Is(err, ParseError{}) {
+				if errors.Is(err, &ParseError{}) {
```

Comment:
does this work? it seems more like an 'As' situation then testing against a sentinel error with `Is`. If it works it's fine, just looks strange to my eye

Reply:
I replaced it with `errors.As`